### PR TITLE
Add performance and syntax improvements to the tip

### PR DESCRIPTION
**Pull request by davegomez**
Changes based on the tip comments and the @nikshulipa suggestions. #192

### CODE REVIEW

File: _posts/en/2016-01-26-filtering-and-sorting-a-list-of-strings.md [hunk 1]
```diff
@@ -54,3 +48,5 @@ console.log(filteredAndSortedKeywords);
 
 // ['abstract', 'arguments', 'await', 'boolean', 'break', 'byte', 'case', 'catch', 'char', 'class', 'const', 'continue', 'debugger', 'default', 'delete', 'do', 'double', 'else', 'enum', 'eval', 'export', 'extends', 'false', 'final', 'finally', 'float', 'for', 'function', 'goto', 'if', 'implements', 'import', 'in', 'instanceof', 'int', 'interface', 'let', 'long', 'native', 'new', 'null', 'package', 'private', 'protected', 'public', 'return', 'short', 'static', 'super', 'switch', 'synchronized', 'this', 'throw', 'throws', 'transient', 'true', 'try', 'typeof', 'var', 'void', 'volatile', 'while', 'with', 'yield']
 ```
+
+*Thanks to everyone for the comments and suggestions!*
```
Comment (loverajoel) [inline _posts/en/2016-01-26-filtering-and-sorting-a-list-of-strings.md]: maybe you can mention and link to his profiles people that suggest you.
Use this format pleas

> Thanks to @blabla @blabla for the comments and suggestions!

**Comment by davegomez** [comment]
ping!

**Comment by davegomez** [comment]
@loverajoel ping

**Comment by loverajoel** [comment]
@davecan sorry

**Pull request merged**